Emit errors when ps is null

richardkazuomiller opened the pull request:
Cleaned up #8.

richardkazuomiller committed:
emit error when ps is undefined

richardkazuomiller committed:
bubble error from client through channel

cromatox commented:
Hi! I use Node-RED for home automation with a Chromecast library (https://github.com/i8beef/node-red-contrib-castv2) which is based on node-castv2. A "Cannot read property 'send' of null" kills my Node-RED server every couple of hours. The code changes solved the problem for me. Is there a chance that the request will be merged?

richardkazuomiller committed:
chore: fix code styles

richardkazuomiller committed:
chore: fix code styles

richardkazuomiller committed:
chore: fix code styles

richardkazuomiller committed:
chore: remove error event bubble

richardkazuomiller commented:
@thibauts I was cleaning up my PRs tab and found this ... it's so old I don't remember what it is but I think I addressed your comments. Please merge or close if you want 🙇‍♂️
